templates/field--field-funding-contact-name.html.twig

```diff
@@ -0,0 +1,7 @@
+<div class="funding-ct--summary-section--contact-info-name">
+    {% for item in items %}
+        <span class="wdn-icon-user" aria-hidden="true"></span><span class="wdn-text-hidden">contact icon</span>
```

Comment:
The thing is, if the field is changed to allow more than one value it will appear broken because the template doesn't render the additional values. I think in all cases the for loop should be used.